dist/stat_data.py

```diff
@@ -800,6 +821,34 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_update_bytes', 'update key and value bytes', 'size'),
     CursorStat('cursor_update_bytes_changed', 'update value size change', 'size'),
 
+    ##########################################
+    # History statistics
```

Comment:
@agorrod, I am curious about prefix hs_*. My first impression when reading through the files was that these have something to do with history store (hs) only realizing afterwards that I made an incorrect assumption.

Reply:
Sure. I chose hs - because I think of timestamps and transaction IDs as forming the history, so it seemed natural to put these in the History statistics section and `hs` was the common prefix there. I'm not wedded to the prefix - feel free to suggest something different.

Reply:
+1.

Reply:
I would personally like them to be under reconciliation stats. Perhaps something like this will make be appropriate:

HistoryStat('rec_internal_pages_with_start_durable_ts', 'Internal pages written that include a newest start durable timestamp '),
HistoryStat('rec_leaf_pages_with_start_durable_ts', 'leaf pages written that included at least one start durable timestamp),
HistoryStat('rec_records_with_start_durable_ts', 'records written that included a start durable timestamp'),

Note that this statistics have a wider audience (e.g., TSEs) and it maybe easier to explain "internal" pages vs. "address" pages.